pkg/renderer/sgml/blank_line.go

```diff
@@ -0,0 +1,29 @@
+package sgml
```

Comment:
would it be possible to make git aware that this file moved from `pkg/renderer/html5` to `pkg/renderer/sgml/html5`, so we don't loose its history? It seems like a handful of those files are deleted in the commit, but not all of them

Reply:
I tried to do that as much as possible, but it was mostly fixing up by doing some work -- originally I developed this by copying the entire tree and modifying.  Then to make the history right I did a backup, git mv, and copy over.  But it looks like I missed a couple.  It's kind of a pain, but if it's important to correct this for a few of those files, then I will see if I can go back and do it.

Reply:
yes, I'd like to keep the history clean. I believe it's doable by moving the files back to their older location, commit, then move again and commit again. If it's too much pain for you, I can take a look and contribute to your PR.

Reply:
I tried a few times with this, and the combination of the rename and content change is enough for git decide that its effectively a deletion and addition.   To test I was to git mv, then I copied over file from another copy of the repo (using just cp, not git).  If you're able to figure out how to preserve the history, then please go ahead.  (You might be able to make this work for some of the files, but I couldn't make it work generally.)

Reply:
ok, I'll give it a try then ;)

Reply:
Resolving this, as you tried and had the same results I did.